src/xref2/env.ml

```diff
@@ -591,24 +575,24 @@ let add_functor_args : Odoc_model.Paths.Identifier.Signature.t -> t -> t =
     in
     match id with
     | (`Module _ | `Result _ | `Parameter _) as mid -> (
-        let m = lookup_module mid env in
-        match m.Component.Module.type_ with
-        | Alias _ -> env
-        | ModuleType e ->
+        match lookup_module mid env with
+        | None -> None
```

Comment:
Should we ignore the lookup failure here and simply return `env` ?

Reply:
Not yet, I think returning `None` is right here - we can choose in `compile.ml` to carry on anyway if we're not in strict mode.